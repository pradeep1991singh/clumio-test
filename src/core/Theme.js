import React from "react";

const Theme = ({ children }) => {
  return (
    <main className="bg-white dark:bg-gray-900 transition-all">{children}</main>
  );
};

export default Theme;
