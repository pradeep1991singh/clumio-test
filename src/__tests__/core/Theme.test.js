import React from "react";
import { render } from "@testing-library/react";
import Theme from "../../core/Theme";

describe("<Theme />", () => {
  test("Snapshot", () => {
    const { asFragment } = render(
      <Theme>
        <span>test children</span>
      </Theme>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});