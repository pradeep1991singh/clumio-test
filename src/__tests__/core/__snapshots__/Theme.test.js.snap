// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Theme /> Snapshot 1`] = `
<DocumentFragment>
  <main
    class="bg-white dark:bg-gray-900 transition-all"
  >
    <span>
      test children
    </span>
  </main>
</DocumentFragment>
`;
